### ISSUE TITLE
Fresh install of ~2.4.1+ crashes after launch on macOS

### ISSUE THREAD

**Issue by tomtastic** [Col-E/Recaf#296]
When no `~/Library/Preferences/Recaf/config/backend.json` exists, or when the file contains `"backend.firsttime": true`, Recaf launches then crashes a few seconds later. 

**To Reproduce**

Tested with;
macOS 11.0 Beta (20A5384c)
Oracle JDK 1.8.0_261, OpenJDK 11.0.8, OpenJDK 14.0.1}

Remove `~/Library/Preferences/Recaf/config/backend.json`, or set contents to include `"backend.firsttime": true`, launch Recaf (from at least version 2.4.1 up to current version 2.11.0) on macOS.

**Exception**
```
$ which jdb
/usr/local/opt/openjdk/bin/jdb
$ which java
/usr/local/opt/openjdk/bin/java
$ java -version
openjdk version "14.0.1" 2020-04-14
OpenJDK Runtime Environment (build 14.0.1+14)
OpenJDK 64-Bit Server VM (build 14.0.1+14, mixed mode, sharing)
$ rm -r ~/.openjfx ~/Library/Preferences/Recaf
$ shasum -a 256 recaf-2.11.0-J8-jar-with-dependencies.jar
fef13ee3eb867e12219cc7e257bc08fb54444ad38a7aa77923d2979ce26eb347  recaf-2.11.0-J8-jar-with-dependencies.jar
$ jdb -Dprism.verbose=true -Djavafx.verbose=true -classpath recaf-2.11.0-J8-jar-with-dependencies.jar
```
https://pastebin.com/1EuvDCTh

**Comment by tomtastic** [comment]
With the latest version 2.11.0, if I have no existing `backend.json`, the app will crash consistently after clicking Cancel to the initial 'This is your first time using Recaf' popup dialog.

If I create a `backend.json` file with just `"backend.firsttime": false` in it, then the app opens fine, no problems.

Now, when 2.11.0 is first run, it crashes without first making a `backend.json` file, and so always crashes on future executions.
However, when earlier version like 2.4.1 / 2.5.0 are first run, they will crash **after** first making a `backend.json` file (with `"backend.firsttime": false`), so future executions work fine.

**Comment by Col-E** [comment]
Point of interest for the first-time crash:

https://github.com/Col-E/Recaf/blob/1dc72b8eafcfa7e72d02142643aded32056ab974/src/main/java/me/coley/recaf/command/impl/Initializer.java#L74

**Comment by tomtastic** [comment]
Not sure if it's any more help but here's the (slightly truncated due to pastebin limits) jdb output with `-verbose`
https://pastebin.com/yCbBtmpb

**Comment by Col-E** [comment]
Fixed in [2.11.1](https://github.com/Col-E/Recaf/releases/tag/2.11.1) 👍
